DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetLobby.cs

```diff
@@ -1049,6 +1198,19 @@ private void DdCurrentChannel_SelectedIndexChanged(object sender, EventArgs e)
             }
         }
 
+        private void VerifyUserList(object sender, EventArgs e)
+        {
+            currentChatChannel.UserListReceived -= VerifyUserList;
+                
+            if (verifyAccountCancellation == null)
+            {
+                verifyAccountCancellation = new CancellationTokenSource();
+
+                CnCNetVerifyAccountsTask.VerifyCall += CnCNetVerifyAccountsTask_VerifyCall;
```

Comment:
If the channel is switched multiple times and multiple user lists are received, is this subscribed to multiple times?